### PR TITLE
Rename README to README.md

### PR DESCRIPTION
**Pull request by kori**
The file is Markdown, so make it be parsed as such.

Fix the table so it parses correctly.

### CODE REVIEW

**Comment by rofl0r** [comment]
there are 2 reasons i didn't merge this PR yet:
- the inclusion of the URLs directly in the text makes it ugly to view in a text editor since the largest line is now much > than 80 chars
- the line wrapping mode of githubs md renderer puts lines that were intended to be on separate lines together in a "wrap on page boundary manner". example:
"without widechar support, etc. this makes it hard to guess which files to link against and which headers to"
^ here the intention was that "this makes it"... goes onto a new line.

if those issues can be addressed and the outcome is pretty/readable in both text editor/terminal and md viewer, then "why not".

**Comment by kori** [comment]
Alright... let's see what I can do.

**Comment by kori** [comment]
There's probably more fixes to be made, but what do you think?

File: README.md [hunk 1]
```diff
@@ -39,29 +39,31 @@ Why do we need an ncurses replacement ?
...
-  use when building a package against ncurses.
-  -lncurses ? -lncursesw -lterminfo -ltic ? curses.h ? ncurses/ncursesw.h ?
-  this filename chaos is often fixed in a distro-specific manner [1].
-  to accomodate for this, ncurses ships its own config tool ncurses(w)5-config
-  instead of a standardized pkg-config description file (disabled by default)
-  to query the necessary CFLAGS and LDFLAGS.
-  unfortunately like every other homebrewn pkg-config replacement, this config
-  utility was designed without cross-compilation in mind, so almost any package
-  using ncurses and autoconf fails to cross-compile when unpatched.
+  - this makes it hard to guess which files to link against and which headers to
```
Comment (rofl0r) [inline README.md]: is there no other way than inserting a new bullet to get the line feed ? :(

**Comment by rofl0r** [comment]
overall looks much better; i like the way you solved the URL issue.
i hope you can find a decent way to force the linefeeds.

File: README.md (hunk 1 above)
Comment (kori) [inline README.md]: I tried different ways and I couldn't figure that out.

Comment (rofl0r) [inline README.md]: ok, in that case it's better to leave that part as it was.
at least it comes out as intended when viewed in a terminal...

**Comment by kori** [comment]
I figured it out!

**Comment by rofl0r** [comment]
thanks, merged manually

**Pull request closed without merging**